ERROR - 2021-11-24 04:29:38 --> ErrorException: pg_query(): Query failed: ERROR:  relation "users_username" already exists in /data/docker/codeigniter4/easyAPI/vendor/codeigniter4/framework/system/Database/Postgre/Connection.php:136
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler()
#1 /data/docker/codeigniter4/easyAPI/vendor/codeigniter4/framework/system/Database/Postgre/Connection.php(136): pg_query()
#2 /data/docker/codeigniter4/easyAPI/vendor/codeigniter4/framework/system/Database/BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute()
#3 /data/docker/codeigniter4/easyAPI/vendor/codeigniter4/framework/system/Database/BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery()
#4 /data/docker/codeigniter4/easyAPI/vendor/codeigniter4/framework/system/Database/Forge.php(521): CodeIgniter\Database\BaseConnection->query()
#5 /data/docker/codeigniter4/easyAPI/vendor/lonnieezell/codigniter-shield/src/Database/Migrations/2020-12-28-223112_create_auth_tables.php(65): CodeIgniter\Database\Forge->createTable()
#6 /data/docker/codeigniter4/easyAPI/vendor/codeigniter4/framework/system/Database/MigrationRunner.php(863): Sparks\Shield\Database\Migrations\CreateAuthTables->up()
#7 /data/docker/codeigniter4/easyAPI/vendor/codeigniter4/framework/system/Database/MigrationRunner.php(186): CodeIgniter\Database\MigrationRunner->migrate()
#8 /data/docker/codeigniter4/easyAPI/vendor/codeigniter4/framework/system/Commands/Database/Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest()
#9 /data/docker/codeigniter4/easyAPI/vendor/codeigniter4/framework/system/CLI/Commands.php(63): CodeIgniter\Commands\Database\Migrate->run()
#10 /data/docker/codeigniter4/easyAPI/vendor/codeigniter4/framework/system/CLI/CommandRunner.php(70): CodeIgniter\CLI\Commands->run()
#11 /data/docker/codeigniter4/easyAPI/vendor/codeigniter4/framework/system/CLI/CommandRunner.php(56): CodeIgniter\CLI\CommandRunner->index()
#12 /data/docker/codeigniter4/easyAPI/vendor/codeigniter4/framework/system/CodeIgniter.php(822): CodeIgniter\CLI\CommandRunner->_remap()
#13 /data/docker/codeigniter4/easyAPI/vendor/codeigniter4/framework/system/CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController()
#14 /data/docker/codeigniter4/easyAPI/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest()
#15 /data/docker/codeigniter4/easyAPI/vendor/codeigniter4/framework/system/CLI/Console.php(48): CodeIgniter\CodeIgniter->run()
#16 /data/docker/codeigniter4/easyAPI/spark(65): CodeIgniter\CLI\Console->run()
#17 {main}
ERROR - 2021-11-24 04:30:32 --> ErrorException: pg_query(): Query failed: ERROR:  relation "users_username" already exists in /data/docker/codeigniter4/easyAPI/vendor/codeigniter4/framework/system/Database/Postgre/Connection.php:136
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler()
#1 /data/docker/codeigniter4/easyAPI/vendor/codeigniter4/framework/system/Database/Postgre/Connection.php(136): pg_query()
#2 /data/docker/codeigniter4/easyAPI/vendor/codeigniter4/framework/system/Database/BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute()
#3 /data/docker/codeigniter4/easyAPI/vendor/codeigniter4/framework/system/Database/BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery()
#4 /data/docker/codeigniter4/easyAPI/vendor/codeigniter4/framework/system/Database/Forge.php(521): CodeIgniter\Database\BaseConnection->query()
#5 /data/docker/codeigniter4/easyAPI/vendor/lonnieezell/codigniter-shield/src/Database/Migrations/2020-12-28-223112_create_auth_tables.php(65): CodeIgniter\Database\Forge->createTable()
#6 /data/docker/codeigniter4/easyAPI/vendor/codeigniter4/framework/system/Database/MigrationRunner.php(863): Sparks\Shield\Database\Migrations\CreateAuthTables->up()
#7 /data/docker/codeigniter4/easyAPI/vendor/codeigniter4/framework/system/Database/MigrationRunner.php(186): CodeIgniter\Database\MigrationRunner->migrate()
#8 /data/docker/codeigniter4/easyAPI/vendor/codeigniter4/framework/system/Commands/Database/Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest()
#9 /data/docker/codeigniter4/easyAPI/vendor/codeigniter4/framework/system/CLI/Commands.php(63): CodeIgniter\Commands\Database\Migrate->run()
#10 /data/docker/codeigniter4/easyAPI/vendor/codeigniter4/framework/system/CLI/BaseCommand.php(112): CodeIgniter\CLI\Commands->run()
#11 /data/docker/codeigniter4/easyAPI/vendor/codeigniter4/framework/system/Commands/Database/MigrateRefresh.php(85): CodeIgniter\CLI\BaseCommand->call()
#12 /data/docker/codeigniter4/easyAPI/vendor/codeigniter4/framework/system/CLI/Commands.php(63): CodeIgniter\Commands\Database\MigrateRefresh->run()
#13 /data/docker/codeigniter4/easyAPI/vendor/codeigniter4/framework/system/CLI/CommandRunner.php(70): CodeIgniter\CLI\Commands->run()
#14 /data/docker/codeigniter4/easyAPI/vendor/codeigniter4/framework/system/CLI/CommandRunner.php(56): CodeIgniter\CLI\CommandRunner->index()
#15 /data/docker/codeigniter4/easyAPI/vendor/codeigniter4/framework/system/CodeIgniter.php(822): CodeIgniter\CLI\CommandRunner->_remap()
#16 /data/docker/codeigniter4/easyAPI/vendor/codeigniter4/framework/system/CodeIgniter.php(410): CodeIgniter\CodeIgniter->runController()
#17 /data/docker/codeigniter4/easyAPI/vendor/codeigniter4/framework/system/CodeIgniter.php(318): CodeIgniter\CodeIgniter->handleRequest()
#18 /data/docker/codeigniter4/easyAPI/vendor/codeigniter4/framework/system/CLI/Console.php(48): CodeIgniter\CodeIgniter->run()
#19 /data/docker/codeigniter4/easyAPI/spark(65): CodeIgniter\CLI\Console->run()
#20 {main}
